taxi/models.py

```diff
@@ -15,7 +17,11 @@ def __str__(self):
 
 
 class Driver(AbstractUser):
-    license_number = models.CharField(max_length=255, unique=True)
+    license_number = models.CharField(
+        max_length=255,
+        unique=True,
+        validators=[RegexValidator(r"\A[A-Z]{3}[0-9]{5}\Z"), ]
+    )
```

Comment:
provide a clear error message for customers when using regular expressions for validation is a good practice